Cargo.toml

```diff
@@ -160,8 +160,8 @@ tokio-socks = { version = "0.5.1", optional = true }
 hickory-resolver = { version = "0.24", optional = true, features = ["tokio-runtime"] }
 
 # HTTP/3 experimental support
-h3 = { version = "0.0.5", optional = true }
-h3-quinn = { version = "0.0.6", optional = true }
+h3 = { git = "https://github.com/Ruben2424/h3.git", branch = "reqwest-sync", optional = true}
```

Comment:
I think you need to contribute this to upstream, crates with git dependencies cannot be published on crates.io and is mutable

Reply:
I already opened a PR. Once merged we also need a new release of h3-quinn

Reply:
Thank you!

Reply:
PR is merged and new versions are published.